cmd/internal/planetscale_edge_database.go

```diff
@@ -39,9 +40,25 @@ type PlanetScaleEdgeDatabase struct {
 }
 
 func (p PlanetScaleEdgeDatabase) CanConnect(ctx context.Context, psc PlanetScaleSource) error {
+	if err := p.isEdgePassword(ctx, psc); err != nil {
+		return errors.New("Unable to reach connect enabled query endpoint for PlanetScale. Please ensure that your organization is enrolled in the Connect beta.")
```

Comment:
```suggestion
		return errors.New("This password will not function with PlanetScale Connect. Please ensure that your organization is enrolled in the Connect beta. ")
```

Reply:
Not convinced at this, but I think we can have stronger words. Even link to the docs?